apps/cookbook/src/app/showcase/accordion-showcase/accordion-showcase.component.html

```diff
@@ -1,54 +1,71 @@
+    other components depend on the package it is not bundled with Kirby for performance reasons and
+    should instead be provided in the application as detailed in the Angular docs for
+    <a href="https://angular.dev/guide/animations#enabling-the-animations-module">
+      enabling the animations module
+    </a>
+    .
   </p>
+  <h2>Simple accordion</h2>
   <cookbook-example-viewer [html]="accordionDefaultExample.template">
     <cookbook-accordion-default-example
       #accordionDefaultExample
     ></cookbook-accordion-default-example>
   </cookbook-example-viewer>
 
-  <h2>Accordion with an initially expanded item</h2>
+  <h3>With initially expanded item</h3>
+  <p>
+    It is possible to have an accordion expanded initially to set focus on a specific information.
+  </p>
   <cookbook-example-viewer [html]="accordionExpandedExample.template">
     <cookbook-accordion-expanded-example
       #accordionExpandedExample
     ></cookbook-accordion-expanded-example>
   </cookbook-example-viewer>
 
   <h2>Accordion inside a card</h2>
+  <p>
+    Accordions may be placed within a card without further adjustments needed to make it stand out
+    from the canvas.
+  </p>
   <cookbook-example-viewer [html]="accordionCardExample.template">
     <cookbook-accordion-card-example #accordionCardExample></cookbook-accordion-card-example>
   </cookbook-example-viewer>
 
-  <h2>Accordion inside card with items</h2>
+  <h2>Accordion with list</h2>
+  <p>
+    Placing a list within an accordion item will automatically adapt spacing and strip the necessary
+    list-styling to ensure that the list looks correct without further adjustments.
+  </p>
+  <cookbook-example-viewer [html]="accordionWithListExample.template">
+    <cookbook-accordion-with-list-example
+      #accordionWithListExample
+    ></cookbook-accordion-with-list-example>
+  </cookbook-example-viewer>
+
+  <h3>Additional styling</h3>
   <p>
-    Since kirby-items come with their own built-in padding the accordions below have
-    <code>hasPadding</code>
-    set to
-    <code>false</code>
-    to avoid double padding. This allows for content inside items to be aligned with the accordion header.
+    Similarly to the simple accordion, the accordion with a list can also be wrapped in a card to
+    make it stand out from the canvas.
   </p>
-  <cookbook-example-viewer [html]="accordionWithListItemsExample.template">
-    <cookbook-accordion-with-items-example
-      #accordionWithListItemsExample
-    ></cookbook-accordion-with-items-example>
+  <cookbook-example-viewer [html]="accordionWithListInCardExample.template">
+    <cookbook-accordion-with-list-in-card-example
+      #accordionWithListInCardExample
+    ></cookbook-accordion-with-list-in-card-example>
   </cookbook-example-viewer>
 
   <h2>Accordion toggle event</h2>
+  <p>
+    A toggle event is emitted whenever an accordion is expanded or collapsed allowing for further
+    interactions.
```

Comment:
Consider simplifying a bit.
```suggestion
    A toggle event is emitted whenever an accordion is expanded or collapsed.
```